### PR TITLE
Added a section about EtherNet/IP incompatibility

### PR DESCRIPTION
**Pull request by fmauch**
As this came up in #191 and #195 we decided to add this to the troubleshooting section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +230,10 @@ Make sure, the IP address setup is correct, as described in the setup guides ([C
 **Note:** This error can also show up, when the ROS driver is not running.
 
 ### When starting the program on the TP, I get a `C207A0` error.
+**Note:** With the current driver version this issue can only happen when the fieldbus is enabled
+*after* the ROS driver has been started. Otherwise you will run into #204 when starting the driver
```
Comment (gavanderhoorn) [inline README.md:234]: This doesn't render as a link for me (ie: the `#204` doesn't link me to the issue).

File: README.md [hunk 2]
```diff
@@ -241,6 +245,12 @@ to the fieldbus scanner can be established (note: This is only to get the
 the fieldbus scanner can indeed be made). If you don't use EtherNet/IP
 fieldbusses at all, you can disable it in the same installation screen. 
 
+### When starting the driver, it crashes with `Variable 'speed_slider_mask' is currently controlled by another RTDE client`
+Probably, you are running into #204. Currently, this driver cannot be used together with an enabled
+EtherNet/IP fieldbus. Disable EtherNet/IP to workaround this error. #204 contains a guide how to do
+this.
```
Comment (gavanderhoorn) [inline README.md:251]: Same here.

File: README.md (hunk 1 above)
Comment (fmauch) [reply README.md]: Resolved in 718a608

File: README.md (hunk 2 above)
Comment (fmauch) [reply README.md]: Resolved in 718a608

File: README.md [hunk 3]
```diff
@@ -230,6 +230,10 @@ Make sure, the IP address setup is correct, as described in the setup guides ([C
 **Note:** This error can also show up, when the ROS driver is not running.
 
 ### When starting the program on the TP, I get a `C207A0` error.
+**Note:** With the current driver version this issue can only happen when the fieldbus is enabled
+*after* the ROS driver has been started. Otherwise you will run into [#204](../../issues/204) when starting the driver
+with an enabled EtherNet/IP fieldbus
```
Comment (gavanderhoorn) [inline README.md:235]:
```suggestion
with an enabled EtherNet/IP fieldbus.
```

**Pull request merged**